internal/releasesources/config.go

```diff
@@ -0,0 +1,7 @@
+package releasesources
+
+type Config struct {
+	OpenshiftMajorVersion           string `envconfig:"OPENSHIFT_MAJOR_VERSION" default:"4"`
```

Comment:
This means that I will need to make an API request for every release to get its support level, do we prefer that instead of a configuration option are just hard coding it ?

Reply:
Or at least one for each major, which is basically 1, so it is better I will remove it

Reply:
For each major, yes
So it's still 1 API request